更新原生平台相关 (#1544)
cocos-creator/creator-docs: en/release-notes/upgrade-guide-v3.0.md

#### Native

The release package generated by Cocos Creator 2.4.3 after building the __Windows__ platform is as follows:

![image](v243-windows.png)

> The release package generated by Cocos Creator 3.0 after building the __Windows__ platform is as follows:

Reviewer: ```suggestion
The release package generated by Cocos Creator v3.0 after building the __Windows__ platform is as follows:
```

Author: It should be "Cocos Creator 3.0" or "v3.0",we have previously discussed and determined in this pr: https://github.com/cocos-creator/creator-docs/pull/1401

![image](v3-windows.png)

> As you can see from the above two figures, there is a big difference between v2.4.3 and v3.0 in the release package generated after building the __Windows__ platform.

Reviewer: ```suggestion
Notice from the above two figures, there is a big difference between v2.4.3 and v3.0 in the release package generated after building the __Windows__ platform.
```
